doc/api/events.markdown

```diff
@@ -137,10 +137,17 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+* `type` {String} The type of event
```

Comment:
No worries. Somehow it was placed as a comment in old code segment. Had me confused for a bit.